frontend/src/components/WidgetContainer.js

```diff
@@ -53,6 +61,12 @@ export default function WidgetContainer(
         ) : null}
         enableCheckboxes={enableCheckboxes}
         exportRows={exportRows}
+        titleDrawerText={titleDrawerText}
+        titleDrawerTitle={titleDrawerTitle}
+        titleDrawerCssClass={titleDrawerCssClass}
+        subtitleDrawerLinkText={subtitleDrawerLinkText}
+        subtitleDrawerLinkTitle={subtitleDrawerLinkTitle}
+        subtitleDrawerCssClass={subtitleDrawerCssClass}

```

Comment:
This is fine but I feel like we may want to refactor to just pass the drawer component and button in rather than all these individual props

Reply:
not necessary on this PR though